site/dev/lint.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
requires 
```
./dev/setup_env.sh
```
here

otherwise, this errors

```
➜  site git:(lint_doc_files) make lint                                                     
dev/lint.sh
 --> fix markdown file styles
/Users/kevinliu/.pyenv/versions/3.12.11/bin/python3: No module named pymarkdown
make: *** [lint] Error 1
```